e2-studio-star-rx72n-firmware/tests/test_bms_monitoring_task.c

```
@@ -69,27 +95,72 @@ void tearDown(void)
+ * @details
+ * Verifies that bms_monitor_task_create() returns k_rx_err_invalid_arg
+ * when soc_critical_pct >= soc_warning_pct (constraint violation).
+ */
+void test_bms_task_create_invalid_thresholds(void)
+{
+  rx_err_t err;
+
+  mock_tx_set_thread_create_return(TX_SUCCESS);
+
+  /* critical >= warning is invalid */
+  const bms_monitor_config_t bad_config = {
```

Clang (14.0.6)

[warning] variable 'bad_config' is not initialized

(cppcoreguidelines-init-variables)